lms/templates/video.html

```diff
@@ -61,30 +62,34 @@ <h4 class="hd hd-5">${_('Video')}</h4>
                     <br>
                 % endif
                 % if public_video_url:
+                    <span class="icon fa fa-share-alt" aria-hidden="true"></span>
                     ${_('Share on:')}
                     <a
                         class="btn-link"
                         id="twitter-share-button"
                         href="${public_video_url}"
                         target="_blank"
                     >
-                        ${_('Twitter')}
+                        <span class="icon fa fa-twitter-square" aria-hidden="true"></span>
+                        <span class="sr">${_('Share on Twitter')}</span>
```

Comment:
Just curious, what's the reasoning for doing this rather than aria_label on the icon?

Reply:
This was just the pattern I saw elsewhere and that Jeff Witt suggested. Will ask if there's any other reasoning,